src/main/kotlin/stringcalculator/Application.kt

```diff
@@ -0,0 +1,47 @@
+package stringcalculator
+
+import stringcalculator.domain.Calculator
+import stringcalculator.ui.InputView
+
+fun main() {
+    Application().run()
+}
+
+private const val INDEX_OF_NUMBER = 0
+
+private const val INDEX_OF_SIGN = 1
+
+class Application {
+
+    fun run() {
+        val inputFormula = InputView().inputFormula()
+        val result = calculate(inputFormula)
+        print(result)
+    }
+
+    private fun calculate(inputFormula: List<String>): Int {
+        val numbers = getNumbers(inputFormula)
+        val signs = getSigns(inputFormula)
+
+        return calculateInternal(numbers, signs)
+    }
+
+    private fun calculateInternal(numbers: List<Int>, signs: List<String>): Int {
+        var summary = numbers[0]
+        for (i in signs.indices) {
+            val calculator = Calculator.of(signs[i])
+
+            val left = summary
+            val right = numbers[i + 1]
+            summary = calculator.calculate(left, right)
+        }
+
+        return summary
+    }
+
+    private fun getSigns(stringFormula: List<String>) =
+        stringFormula.filterIndexed { index, _ -> index % 2 == INDEX_OF_SIGN }.map { it }
+
+    private fun getNumbers(stringFormula: List<String>) =
+        stringFormula.filterIndexed { index, _ -> index % 2 == INDEX_OF_NUMBER }.map { it.toInt() }
```

Comment:
메서드가 호출되는 순서대로 나열하면 코드의 가독성이 올라갈 것 같아요. 😆

```suggestion
    private fun getNumbers(stringFormula: List<String>) =
        stringFormula.filterIndexed { index, _ -> index % 2 == INDEX_OF_NUMBER }.map { it.toInt() }
        
    private fun getSigns(stringFormula: List<String>) =
        stringFormula.filterIndexed { index, _ -> index % 2 == INDEX_OF_SIGN }.map { it }
        
    private fun calculateInternal(numbers: List<Int>, signs: List<String>): Int {
        var summary = numbers[0]
        for (i in signs.indices) {
            val calculator = Calculator.of(signs[i])

            val left = summary
            val right = numbers[i + 1]
            summary = calculator.calculate(left, right)
        }

        return summary
    }
```

Reply:
조금 급한 마음에 메서드 순서를 정리 못하고 커밋을 했네요 😃 
피드백 감사합니다!